build/device.mk

```diff
@@ -272,6 +272,16 @@ PRODUCT_COPY_FILES += \
     vendor/xen/sensors/cfg/sensors-config.json:$(TARGET_COPY_OUT_VENDOR)/etc/vehicle/sensors-config.json
 
 # Exterior View System (EVS)
+USE_CARLA_EVS=true
+ifeq ($(USE_CARLA_EVS), true)
+ENABLE_CAMERA_SERVICE=true
+ENABLE_EVS_SAMPLE=true
+$(call inherit-product, vendor/xen/evs/evs.mk)
+
+PRODUCT_PROPERTY_OVERRIDES += \
+	persist.vendor.evs.uri="ws://192.168.122.1"
```

Comment:
I suppose here we need to use a host name(like we did previously with wwwivi) instead of IP. Since our setup configuration may change and will resolve host using static record in /etc/hosts

Reply:
Not sure if we need to add carla host to /etc/hosts

Reply:
as was agreed with @arminn flag was removed